internal/postprocessor/main.go

```diff
@@ -150,6 +149,13 @@ func (c *config) run(ctx context.Context) error {
 		log.Println("exiting post processing early")
 		return nil
 	}
+	// TODO(codyoss): In the future we may want to make it possible to be able
+	// to run this locally with a user defined remote branch.
+	modules, err := c.AmendPRDescription(ctx)
+	if err != nil {
+		return err
+	}
+	c.modules = append(c.modules, modules...)
 	if err := gocmd.ModTidyAll(c.googleCloudDir); err != nil {
 		return err
 	}
```

Comment:
Do we want to add a check to see if c.modules is empty, and if so, mod tidy all?

Reply:
I think if empty we should detect what code to run for. See other comments.